sources/packages/api/src/services/education-program/education-program.service.ts

```diff
@@ -208,8 +221,12 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
     } else {
       // default sort and order
       summaryResult.orderBy(
-        sortProgramsColumnMap(DEFAULT_SORT_FIELD),
-        FieldSortOrder.ASC,
+        `CASE programs.approvalStatus
```

Comment:
@ann-aot, @dheepak-aot was talking with me during my last PR to us starting to consider that, when the enum is already ordered accordingly to the business requirement we can directly use the enum column to do the order by.
I believe that the execution on DB will be way better also.

Reply:
Program `approval_status` is a character field its not an enum, maybe we can create a technical debt to  make this field enum
![image](https://user-images.githubusercontent.com/77353155/153956388-3e15fbc7-5684-4f90-bb5a-985530e75d23.png)